careers.html

```diff
@@ -29,12 +29,17 @@
 
         <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
             <div class="navbar-nav ml-auto">
-                <a class="nav-item nav-link" href="#">ABOUT US</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">HISTORY</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">CATALOGUE</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link" href="#">PARTNERS</a><span class="navbar-dot"></span>
-                <a class="nav-item nav-link active" href="#">CAREERS</a>
-                <button type="button" id="contactButton">Contact</button>
+                <a class="nav-item nav-link" href="index.html#homeJumbotronDiv">ABOUT US</a><span
```

Comment:
i think you can just omit `index.html`. just `href="#homeJumbotronDiv"` should be able to work too?

same for the rest below

Reply:
@shaunwan-surialabs just tried it... doesnt work... index.html needs to be there..

Reply:
ok sorry my bad..i thought this was `index.html` but it's actually `career.html`